src/Microsoft.OData.Client/Microsoft.OData.Client.cs

```diff
@@ -12,16 +12,13 @@
 using System.Reflection;
 using System.Resources;
 using System.Threading;
-#if !PORTABLELIB
-using System.Security.Permissions;
-#endif
 
 namespace Microsoft.OData.Client
 {
     /// <summary>
     ///    AutoGenerated resource class. Usage:
     ///
-    ///        string s = TextRes.GetString(TextRes.MyIdentifier);
+    ///        string s = TextRes.GetString(TextRes.MyIdenfitier);

```

Comment:
Why is getting changed from `MyIdentifier` to `MyIdenfitier`?

Reply:
seems like the generator got changed again thanks for noticing this